mapfile.c

```diff
@@ -5414,8 +5449,10 @@ void initQueryMap(queryMapObj *querymap)
   MS_INIT_COLOR(querymap->color, 255,255,0,255); /* yellow */
 }
 
-int loadQueryMap(queryMapObj *querymap)
+int loadQueryMap(queryMapObj *querymap, mapObj *map)
 {
+  querymap->map = (mapObj *)map;
```

Comment:
Could there be any issues with adding the map as a property to the querymap? Would it be safer to pass in the `maxsize` and `minsize` values directly to this function?

Reply:
We use the same approach with other objects (e.g. layer, scalebar) where there is an opaque pointer back to the parent so this should be ok. I was trying to avoid changing function signatures too much.